internal/access/credential.go

```diff
@@ -105,8 +112,11 @@ func updateCredential(c *gin.Context, user *models.Identity, newPassword string,
 	rCtx := GetRequestContext(c)
 	db := rCtx.DBTxn
 
-	err := checkPasswordRequirements(db, newPassword)
-	if err != nil {
+	if err := checkPasswordRequirements(db, newPassword); err != nil {
+		return err
+	}
+
+	if err := checkBadPasswords(newPassword); err != nil {
 		return err
```

Comment:
I feel like we should be proceeding in the case of certain errors here. For example if`ncsc.gov.uk/static-assets/documents/PwnedPasswordsTop100k.txt` ever becomes unavailable or invalid, but it has not been cached locally no one will be able to sign-up with a password.